tests/e2e/actions.go

```diff
@@ -526,6 +456,7 @@ func (tr TestRun) voteGovProposal(
 
 	wg.Wait()
 	time.Sleep(time.Duration(tr.chainConfigs[action.chain].votingWaitTime) * time.Second)
+	tr.waitBlocks(action.chain, 1, 5*time.Second)
```

Comment:
We had cases where the tests were failing although the gov proposal was passing. Specifically, we had a test where the consumer chain was stopped and the proposal passed but the chain remained there. A chain is not removed immediately when a proposal passes but maybe a block later (see [here](https://github.com/cosmos/interchain-security/blob/main/x/ccv/provider/keeper/proposal.go#L372)) and that's why we wait one more block here. (thanks to @p-offtermatt !)